### PR TITLE
C24_WMDE_Desktop_DE_14

### PR DESCRIPTION
**Pull request by Sperling-0**
Ticket: https://phabricator.wikimedia.org/T379950

### CODE REVIEW

File: banners/desktop/C24_WMDE_Desktop_DE_14/content/BannerSlidesVar.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		<p>
+			Dann entscheiden Sie sich, eine der seltenen Ausnahmen zu sein, und geben Sie etwas zurück. Falls Sie zögern,
+			<a
+				id="reasons-to-donate-link"
+				class="wmde-banner-reasons-to-donate-link t-reasons-to-donate-link"
+				@click.prevent="$emit( 'showReasonsToDonate' )"
+				href="#"
+			>
+				hier sind 10 gute Gründe
+			</a>
```
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_14/content/BannerSlidesVar.vue]:
```suggestion
            <button class="wmde-banner-reasons-to-donate-link" @click.prevent="$emit( 'showReasonsToDonate' )">
				hier sind 10 gute Gründe
			</button>
```

File: banners/desktop/C24_WMDE_Desktop_DE_14/content/BannerTextVar.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+				Wissen im Wert einer Tasse Kaffee geschenkt? Dann entscheiden Sie sich, eine der seltenen Ausnahmen zu
+				sein, und geben Sie etwas zurück. Falls Sie zögern,
+				<button
+					id="reasons-to-donate-link"
+					class="wmde-banner-reasons-to-donate-link t-reasons-to-donate-link"
+					@click.prevent="$emit( 'showReasonsToDonate' )"
+					href="#"
+				>
+					hier sind 10 gute Gründe
+				</button>
```
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_14/content/BannerTextVar.vue]:
```suggestion
				<button class="wmde-banner-reasons-to-donate-link" @click.prevent="$emit( 'showReasonsToDonate' )">
					hier sind 10 gute Gründe
				</button>
```

**Review by Abban — COMMENTED** [review]
You should make both of the links buttons, and remove the unused ID and classes and href from them

File: banners/desktop/C24_WMDE_Desktop_DE_14/banner_ctrl.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+} );
+
+app.use( TranslationPlugin, translator );
+app.use( DynamicTextPlugin, {
+	campaignParameters: page.getCampaignParameters(),
+	date,
+	formatters: localeFactory.getFormatters(),
+	impressionCount,
+	translator,
+	urgencyMessageDaysLeft: 45
```
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_14/banner_ctrl.ts]: You should delete this as it's making the urgency message appear too early

File: banners/desktop/C24_WMDE_Desktop_DE_14/banner_var.ts [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+} );
+
+app.use( TranslationPlugin, translator );
+app.use( DynamicTextPlugin, {
+	campaignParameters: page.getCampaignParameters(),
+	date,
+	formatters: localeFactory.getFormatters(),
+	impressionCount,
+	translator,
+	urgencyMessageDaysLeft: 45
```
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_14/banner_var.ts]: And delete this too

**Review by gbirke — APPROVED** [review]

**Pull request merged**